cmake/compiler/host-gcc/target.cmake

```diff
@@ -9,10 +9,19 @@ find_program(CMAKE_RANLILB      ranlib )
 find_program(CMAKE_READELF      readelf)
 find_program(CMAKE_GDB          gdb    )
 
-set(CMAKE_ASM_FLAGS           -m32 )
-set(CMAKE_C_FLAGS             -m32 )
-set(CMAKE_CXX_FLAGS           -m32 )
-set(CMAKE_SHARED_LINKER_FLAGS -m32 ) # unused?
+# -march={pentium,lakemont,...} do not automagically imply -m32, so
+# adding it here.
+
+# There's only one 64bits ARCH (actually: -mx32). Let's exclude it to
```

Comment:
nitpick: Until arm aarch64 is added :)

Reply:
... but not MIPS64? :-)

This is actually why I'm suggesting in the next paragraph to move this -m32 flag entirely out of this file in the longer term and spread them instead next to the corresponding -march flags. @SebastianBoe ?
This would be a significant refactoring though, way beyond the scope of this small regression fix.